skin.scss

```diff
@@ -0,0 +1,35 @@
+.highlight {
+    background-color: #ddd;
+}
+
+button.btn {
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 20px;
+  font-weight: normal;
+  line-height: 1.15;
+  text-align: center;
+  vertical-align: middle;
+}
+btn{ 
+   &--primary {
+    background-color: #ffffff;
+  }
+
+  &--secondary {
+    background-color: #5a1212;
+
+  }
+
+
+  &--large{
+      background-color: #470606;

```

Comment:
You've changed the colour, not the size of the button here. You need to increase the `font-size` and maybe the `padding-left` and `padding-right`.

Reply:
I just changed the color to see if it is gonna to work or not.
still does not work. I am going to send a pull request again.